src/main_app/admin/sidebar.py

```
@@ -46,38 +51,156 @@ def generate_list_item(item: SidebarItem) -> str:
+                        css_class = "active"
+
+                    if not css_class and self.active_route == item.id:
+                        css_class = "active"
+
+                if css_class:
+                    active_group = key
+                    active_id = item.id
+                    break
+
+        if not active_group:
+            active_group = list(self.menu.keys())[0]
```

Ruff (0.15.20)

[warning] Prefer `next(iter(self.menu.keys()))` over single element slice

Replace with `next(iter(self.menu.keys()))`

(RUF015)